### DISCUSSION TITLE
how to use this sketchpad ?

### DISCUSSION THREAD

**Discussion by KshitijSabale** [mbrlabs/Lorien#66] [Q&A] [answered]
I begineer I dont know much about gdscript  .
Can you please tell me how to use this sketch pad on my windows desktop ?
I have cloned the repository but do i need godat engine to run it or there is any single line command to run it ?

**Comment by mbrlabs** [comment, marked as answer]
If you just want to check it out and use it, there are prebuilt binaries for all platforms available on Github: https://github.com/mbrlabs/Lorien/releases/tag/v0.3.0

If you want to run it from source you need to download Godot and import the project. Just clone the repo and select the `lorien/project.godot` file in Godot's project manager (should be clear what to do when you run Godot).
